ports/unix/Makefile

```diff
@@ -178,7 +178,9 @@ ifneq ($(FROZEN_MANIFEST)$(FROZEN_MPY_DIR),)
 # freeze, then invoke make with FROZEN_MANIFEST=manifest.py (be sure to build from scratch).
 CFLAGS += -DMICROPY_QSTR_EXTRA_POOL=mp_qstr_frozen_const_pool
 CFLAGS += -DMICROPY_MODULE_FROZEN_MPY
-CFLAGS += -DMPZ_DIG_SIZE=16 # force 16 bits to work on both 32 and 64 bit archs
+ifneq ($(MICROPY_FORCE_32BIT),1)
```

Comment:
I'm not sure this will catch all build scenarios: eg what about building on a 32-bit machine?  Then this config option may not be set but the dig size should still be 16.

Reply:
Hmm, you're right. I'll think of another way to check the compiler version perhaps.

Without this change, since mpy-tool assumes MPZ_DIG_SIZE=16, you must define MPZ_DIG_SIZE=16 in your CFLAGS/port to have it compile. It better be automatic.

Reply:
If you rebase this PR on to latest master that will include a new Travis build for 32-bit unix, which can test this.

Reply:
I think we should just remove this change.